docker/omsevents/bootstrap.sh

```diff
@@ -1,10 +1,36 @@
 #!/bin/bash
 
-cp  /root/configFile.json /usr/app/oms-events/lib/config/configFile.json
+if [ -f "/usr/scripts/strapstate/omsevents" ]
+then
+	echo "Bootstrap-file found, not executing bootstrap script"
+else
+	echo "Waiting for omscore to finish bootstrapping"
+	while [ ! -f "/usr/scripts/strapstate/omscore" ]
+	do
+	 	sleep 2
+	done
 
-# wait until supervisor has restarted the service
-printf 'Config updated, waiting for service to restart\n'
-sleep 10
+	echo "Starting bootstrap for oms-events"
 
-# Set up the api-key in the configuration file.
-curl -sS http://localhost:8082/registerMicroservice
+	cp  /usr/scripts/configFile.json /usr/app/oms-events/lib/config/configFile.json
+
+	cd /usr/app/oms-events
+	npm install
+
+	# Allow the omsevents container to start
+	touch /usr/scripts/strapstate/omsevents-basic
+
+	echo "Waíting for omsevents to start up"
+	until $(curl --output /dev/null --silent --fail http://omsevents:8082/ping); 
```

Comment:
I don't see why the name resolution should not work in a vagrant, it's part of docker. I don't know if it always was, so maybe you installed an ancient version of it... could you tell me which version you use?